x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_disabled_fields_body.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiTableBody, EuiTableRow, EuiTableRowCell, EuiText, EuiHealth } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ResultSettingsLogic } from '..';
+
+export const ResultSettingsDisabledFieldsBody: React.FC = () => {
+  const { schemaConflicts } = useValues(ResultSettingsLogic);
+  return (
+    <EuiTableBody>
+      {Object.keys(schemaConflicts).map((fieldName) => (
+        <EuiTableRow key={fieldName}>
+          <EuiTableRowCell colSpan={6}>
+            <EuiText size="s" data-test-subj="ResultSettingFieldName">
+              {fieldName}
+            </EuiText>
```

Comment:
last typography suggestion
```suggestion
            <EuiText size="s" data-test-subj="ResultSettingFieldName">
              <code>{fieldName}</code>
            </EuiText>
```